metadb/rqlite/cluster/join.go

```diff
@@ -43,14 +44,14 @@ func Join(srcIP string, joinAddr []string, id, addr string, voter bool, numAttem
 				return j, nil
 			}
 		}
-		logger.Printf("failed to join cluster at %s: %s, sleeping %s before retry", joinAddr, err.Error(), attemptInterval)
+		logger.Errorf("failed to join cluster at %s: %s, sleeping %s before retry", joinAddr, err.Error(), attemptInterval)
 		time.Sleep(attemptInterval)
 	}
-	logger.Printf("failed to join cluster at %s, after %d attempts", joinAddr, numAttempts)
+	logger.Errorf("failed to join cluster at %s, after %d attempts", joinAddr, numAttempts)
```

Comment:
```suggestion
	logger.Warnf("failed to join cluster at %s, after %d attempts", joinAddr, numAttempts)
```

elsewhere it might be worth changing as well

Reply:
i think it's better to keep error here, because after attempts, it will cause the application exit when starting the supernode